scala2java-core/src/test/scala/io/github/effiban/scala2java/core/collectors/TemplateAncestorsCollectorTest.scala

```diff
@@ -171,7 +171,7 @@ class TemplateAncestorsCollectorTest extends UnitTestSuite {
 
     actualAncestorTypeRefMap.size shouldBe 1
     val (actualParentType, actualAncestorTypes) = actualAncestorTypeRefMap.head
-    actualParentType.structure shouldBe t"scala.collection.Iterable".structure
+    actualParentType.structure shouldBe t"scala.Iterable".structure
```

Comment:
This is a side effect of the refactor - the parent will be returned without dealiasing, but that's ok since it's not needed there.